shared/common-adapters/loading-line.native.js

```diff
@@ -26,6 +26,7 @@ class LoadingLine extends Component<Props, {fadeAnim: any}> {
         NativeAnimated.timing(this.state.fadeAnim, {
           duration: 600,
           easing: NativeEasing.ease,
+          isInteraction: false,
```

Comment:
this fixes the issue where if the bar was showing the inbox rows wouldn't update

Reply:
also a nice comment, in case someone fees like changing it back someday for some reason?